docs/zkapps/advanced-snarkyjs/permissions.mdx

```diff
@@ -0,0 +1,294 @@
+    },
+    // ...
+    "body": {
+        // ...
+        "update": {
+            "appState": [
+                "1",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0",
+                "0"
+            ],
+            // ...
+        },
+        // ...
+    }
+}
+```
+
+As you can see, this example account update has an authorization of kind `signature` provided, and you can also see it's trying to update the app state of the smart contract!
+Let's consider a smart contract that has the permission `editState` set to the authorization `none`. 
+This means, since the authorization is `none`, everyone can freely change the state of the smart contract as they please! Obviously, this is not something we likely want. So in order to only allow state changes when a valid proof accompanies the account update (which wants to access said state), we have to set our authorization to `proof`!
+This makes sure that state is only being changed if and only if the user generated a valid proof by executing a smart contract method correctly!
+
+Let's briefly go into detail what each authorization means:
+
+`none`: Everyone has access to fields with permission set to `none` - and therefore can manipulate the fields as they please! 
+
+`impossible`: If a fields permission is set to `impossible`, nothing can ever change this field!  
+
+`signature`: Fields which have their permission set to `signature` can only be manipulated by account updates that are accompanied and authorized by a valid signature.
+
+`proof`: Fields which have their permission set to `proof` can only be manipulated by account updates that are accompanied and authorized by a valid proof. Proofs are generated by proving the execution of a smart contract method.
+
+`proofOrSignature`: As the name might suggest, permissions with authorization set to `proofOrSignature` accept both a valid signature or a valid proof. 
+
+## The Default
+
+Smart contracts, when first deployed, always start with a default set of permissions. 
+Currently, the default is this:
+
+
+`editState`: `proof`
+
+`send`: `signature`
+
+`receive`: `none`
+
+`setDelegate`: `signature`
+
+`setPermissions`: `signature`
+
+`setVerificationKey`: `signature`
+
+`setZkappUri`: `signature`
+
+`editSequenceState`: `proof`
+
+`setTokenSymbol`: `signature`
+
+
+## Example
+
+To better understand how we can leverage permissions to make our smart contract more secure, let's look at a few examples.
+
+### Sending MINA from smart contracts
+
+Some smart contracts will not only manage state, but also token (such as the native MINA token). In order to prevent malicious actors from withdrawing all funds, we can use permissions to secure them.
+
+Consider the following smart contract `UnsecureContract`, which is available [here](https://github.com/o1-labs/snarkyjs/blob/main/src/examples/zkapps/simple_zkapp_payment.ts) as well
```

Comment:
we have lengthy examples that utilize permissions. I wanted to show some code to guide the reader on how to use and define permissions, but i didnt want to write a whole new examples that we would have to maintain, including all boilerplate code etc. thoughts?